test/integration/operator_policy_error_test.go

```diff
@@ -0,0 +1,126 @@
+// Copyright (c) 2022 Red Hat, Inc.
+// Copyright Contributors to the Open Cluster Management project
+
+package integration
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	policiesv1 "open-cluster-management.io/governance-policy-propagator/api/v1"
+
+	"github.com/stolostron/governance-policy-framework/test/common"
+)
+
+var _ = Describe("GRC: [P1][Sev1][policy-grc] Test operatorpolicy errors",
+	Ordered, Label("BVT"), func() {
+		const (
+			policyPath            = "../resources/operator_policy/test-op-err-initial.yaml"
+			correctPolicyPath     = "../resources/operator_policy/test-op-err-correct.yaml"
+			policyName            = "test-op-err-initial"
+			operatorPolicyName    = "test-op-err-initial"
+			subscriptionNamespace = "grcqeoptest-ns-43568"
+		)
+
+		BeforeAll(func() {
+			By("Create policy " + policyName + " which contains " + operatorPolicyName)
+			common.DoCreatePolicyTest(policyPath, common.GvrOperatorPolicy)
+		})
+
+		AfterAll(func() {
+			_, err := common.OcHub(
+				"delete", "-f", policyPath,
+				"-n", userNamespace, "--ignore-not-found=true",
+			)
+			Expect(err).ToNot(HaveOccurred())
+
+			_, err = common.OcManaged(
+				"delete", "ns", subscriptionNamespace,
```

Comment:
In the unlikely event the first test fails, this is also going to fail. It's probably safer to add an `--ignore-not-found` here also.